File: java/MASTER_SLAVE_WHOLE_PART_PROXY/GestorDatosCliente.java
public class GestorDatosCliente {
    //Atributos
    private static DatosCliente datosCliente = new DatosCliente();

    //Constructor
    public GestorDatosCliente(){}

    //Métodos
    static DatosCliente preparaDatosCliente(String nombre, String apellidoP, String apellidoM, int edad, String tipoC){
        datosCliente.setDatosCliente(nombre, apellidoP, apellidoM, edad, tipoC);
        return datosCliente;
    }
}

File: php/MASTER_SLAVE_WHOLE_PART_PROXY/gestores/GestorDatosCliente.php
<?php
require './datos/DatosCliente.php';

class GestorDatosCliente{
    //attr
    
    public static $datosCliente;
    
    public static function init(){
        self::$datosCliente = new DatosCliente(); 
    }

    public static function preparaDatosCliente($nombre, $apellidoP, $apellidoM,$edad, $tipoC){
        self::init();
        self::$datosCliente->setDatosCliente($nombre, $apellidoP, $apellidoM, $edad, $tipoC);
        
        return self::$datosCliente;
    }
}